lntest/harness.go

```diff
@@ -2183,6 +2184,39 @@ func (h *HarnessTest) ReceiveHtlcInterceptor(
 	return nil
 }
 
+// ReceiveInvoiceAcceptor waits until a message is received on the invoice
+// acceptor interceptor stream or the timeout is reached.
+func (h *HarnessTest) ReceiveInvoiceAcceptor(
```

Comment:
rename `InvoiceAcceptor` in a few places in this commit, and in the commit message